android/assets/jsons/translations/French.properties

```diff
@@ -642,11 +642,11 @@ Convert production to gold at a rate of 4 to 1 = Convertit la production en or a
 Convert production to science at a rate of [rate] to 1 = Convertit la production en science au taux de [rate] pour 1
 The city will not produce anything. = La cité ne produira plus rien
  # Requires translation!
-Worked by [cityName] = 
+Worked by [cityName] = Employer par [cityName] 
```

Comment:
What is wrong in this one? Do you have special `i` in your language that looks like normal `i`?